projects/components/src/paginator/paginator.component.scss

```diff
@@ -16,27 +16,82 @@
   }
 
   .page-size-select {
-    min-width: 86px;
+    min-width: $paginator-select-width;
     margin-left: 12px;
   }
 
   .pagination-buttons {
     display: flex;
     margin: 0 24px 0 12px;
+    ::ng-deep {
+      .toggle-button-group {
```

Comment:
I guess all of this is required because the paginator toggle has a slightly different style that toggle button component. @jake-bassett What should we do for this inconsistency?

Reply:
this is way too much reskinning IMO - there's a ton of class dependencies here that could easily change. I get that we sometimes need ng-deep, but at this point, I think using toggle is more likely to introduce breakages in the future and would rather either make the paginator style consistent with toggle or not use toggle in this component and implement the buttons as part of this component.

Reply:
+1 on not reusing a component if we need to modify it so heavily.

Reply:
aah. I kind of suggested to use toggle since the behavior was similar. I didn't know the styling of pagination buttons is so different. Should we instead look into combining the two styles form ux side?

Reply:
Yeah, I can see the draw of using the toggle, but the use cases are just a little different. Pagination is really two buttons visually tied together, but functionality independent. Toggles have one or more active states.